pypesto/objective/jax/base.py

```diff
@@ -214,7 +166,44 @@ def check_sensi_orders(self, sensi_orders, mode: ModeType) -> bool:
         if not self.check_mode(mode):
             return False
         else:
-            return self.base_objective.check_sensi_orders(sensi_orders, mode)
+            return (
+                self.base_objective.check_sensi_orders(sensi_orders, mode)
+                and max(sensi_orders) == 0
+            )
+
+    def __call__(
+        self,
+        x: jnp.ndarray,
+        sensi_orders: Tuple[int, ...] = (0,),
+        mode: ModeType = MODE_FUN,
+        return_dict: bool = False,
+        **kwargs,
+    ) -> Union[jnp.ndarray, Tuple, ResultDict]:
+        """
+        See `ObjectiveBase` for more documentation.
```

Comment:
```suggestion
        See :class:`ObjectiveBase` for more documentation.
```
To create a link to base class?